services/frontend/src/views/admin/News.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <div>
+    <h1>Aktualności</h1>
+    <v-divider />
+    <NewsData v-bind:newsId="this.$route.params.id" />
```

Comment:
if im not mistaken `newsData` doesn't use  newsID

Reply:
You're right, prolly forgot to remove it